### ISSUE TITLE
Is using the Personal Access Token the only way to get the runner token to register the self-hosted runner at the organization level?

### ISSUE THREAD

**Issue by michaelsync** [myoung34/docker-github-actions-runner#74]
Can we use the github app or oauth app instead of github PAT to generate the RUNNER_TOKEN? The problem with PAT is that none of our Github organization admins want to give the PAT. 

````
ACCESS_TOKEN | A github PAT to use to generate RUNNER_TOKEN dynamically at container start. Not using this requires a valid RUNNER_TOKEN

**Comment by myoung34** [comment]
To be honest I don't know. You're welcome to try. If you want to ask in the upstream actions repo and follow back with info I'll update the docs or code with your findings

**Comment by michaelsync** [comment]
Thanks, I am not sure what the upstream actions repo is. But I did ask around here. 

- https://github.community/t/can-we-use-the-github-app-or-oauth-app-instead-of-github-pat-to-generate-the-runner-token/146505
- https://stackoverflow.com/questions/65090500/how-to-use-the-github-app-oauth-app-to-register-the-github-action-self-hosted-ru
- https://github.com/SanderKnape/github-runner/issues/15

I will update you if I got some info. Thanks.
